src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordClone.cs

```diff
@@ -134,25 +127,41 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
             throw ExceptionUtilities.Unreachable;
         }
 
-        internal static MethodSymbol? FindValidCloneMethod(NamedTypeSymbol containingType)
+        internal static MethodSymbol? FindValidCloneMethod(TypeSymbol containingType, ref HashSet<DiagnosticInfo>? useSiteDiagnostics)
```

Comment:
>useSiteDiagnostics [](start = 115, length = 18)

nit: consider commenting or renaming (`baseUseSiteDiagnostics`) to clarify they are for the base type, not for the clone method. Could also be renamed in calling code

Reply:
> consider commenting or renaming (baseUseSiteDiagnostics) to clarify they are for the base type, not for the clone method. Could also be renamed in calling code

I do not believe we want play games like that with names. It is intentionally doesn't matter where the errors are coming from, the point is that they should be reported at the use-site and that is exactly what the name is reflecting. We consistently use this name throughout the code base and intentionally do not try to specify the source/reason for the errors.

---
In reply to: [456676712](https://github.com/dotnet/roslyn/pull/46053#discussion_r456676712) [](ancestors = 456676712)